HomeWorkApp/src/main/java/org/example/AttendanceLogger.java

```diff
@@ -0,0 +1,100 @@
+package org.example;
+
+import java.util.*;
+
+public class AttendanceLogger {
+
+    // Журнал посещений
+    private final List<VisitRecord> visitLog;
+
+    // Конструктор
+    public AttendanceLogger() {
+        this.visitLog = new ArrayList<>();
+    }
+
+    // Метод для регистрации нового посещения
+    public void registerVisit(String userId, String timestamp) {
+        visitLog.add(new VisitRecord(userId, timestamp));
```

Comment:
Лучше бы система самостоятельно выставляла время, а не снаружи.
```suggestion
    public void registerVisit(String userId) {
        visitLog.add(new VisitRecord(userId, timestamp));
```